client/js/lounge.js

```diff
@@ -465,6 +468,22 @@ $(function() {
 		}
 	}
 
+	function updateDisconnectedUsers(channel, message) {
+		switch (message.type) {
+		case "nick":
+			channel.find(".msg .user[data-name='" + message.from + "']").addClass("disconnected");
+			channel.find(".msg .user[data-name='" + message.new_nick + "']").removeClass("disconnected");
+			break;
+		case "join":
+			channel.find(".msg .user[data-name='" + message.from + "']").removeClass("disconnected");
+			break;
+		case "part":
```

Comment:
@xPaw, do you see any performance concerns here or somewhere else in this PR? Something tells me there will be some overhead, similar to the PR that was looking for all nicks in messages.